src/lib/pubkey/dilithium/dilithium.cpp

```diff
@@ -150,6 +71,11 @@ namespace Botan
       {
       }
 
+      // Delete Copy constructor
+      Dilithium_PublicKeyInternal(const Dilithium_PublicKeyInternal&) = delete;
+      // Delete Move Assignemnt constructor
+      Dilithium_PublicKeyInternal(const Dilithium_PublicKeyInternal&&) = delete;
```

Comment:
Move assignment constructor doesn't take a const rvalue reference.

```C++
Dilithium_PublicKeyInternal(Dilithium_PublicKeyInternal&&) = delete
```

also: please adhere to the "rule of five": https://en.cppreference.com/w/cpp/language/rule_of_three
i.e: delete copy and move assignment operators.